test/types.js

```diff
@@ -2,6 +2,15 @@
 var assert = require('assert')
 var ref = require('../')
 
+// a type similar to ref-struct's StructType
+// used for types refType name property test
+var StructType = function() {}
+StructType.prototype = Object.create({}, {
```

Comment:
Heh, I actually had it within the test before but got complaints in node 0.8 about scoped functions. But I think it was due to a different test failing due to --use_strict. Anyways, I just pushed an update that moves the definition back into the test and removes the Object.create and Object.defineProperty. If I still get a failure, I may just back out the --use_strict addition from the test runner for the time being.